handlers/advertisewithus.go

```diff
@@ -66,7 +68,7 @@ func AdvertiseWithUsPost(w http.ResponseWriter, r *http.Request) {
 	// escape html
 	msg = template.HTMLEscapeString(msg)
 
-	err = mail.SendTextMail("support@beaconcha.in", "New ad inquiry", msg, []types.EmailAttachment{})
+	err = mail.SendTextMail("inquiries@beaconcha.in", "New ad inquiry", msg, []types.EmailAttachment{})
```

Comment:
I have discussed this with @Buttaa: Having our own mail address hardcoded within the sources makes the explorer unusable for other people.

Please make this mail address configurable via the config. Please check for other mail addresses too.